test/services/tempMailApi.test.js

```diff
@@ -155,15 +155,15 @@ describe('TempMail', () => {
       }
     });
 
-    it('should throw err if token is not valid', async () => {
+    it('should throw err if token is not valid when gettingMailInbox', async () => {
       axios.post
```

Comment:
You only have 1 post call, but in the test you're mocking twice...?

Reply:
Ive pushed things way to how i did it at first and explained you in private